hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -1226,6 +1226,23 @@ class Emit[C](
         )
         EmitCode(lengthTriplet.setup, lengthTriplet.m, PCode(pt, result))
 
+      case x@StreamLen(a) =>
+        val streamOpt = emitStream(a)
+        val lenOpt = streamOpt.map { ss =>
+          val count = mb.newLocal[Int]("stream_length")
```

Comment:
Minor, but `count` could be declared in the `None` case, so it's clear that's the only place it's used, and so we don't generate an unused local when the length is known.